src/main/java/org/spongepowered/api/statistic/Statistics.java

```diff
@@ -24,7 +24,7 @@
  */
 package org.spongepowered.api.statistic;
 
-import org.spongepowered.api.util.generator.dummy.DummyObjectProvider;
+import static org.spongepowered.api.util.generator.dummy.DummyObjectProvider.createFor;
```

Comment:
IMO its quite obvious where this method comes from (especially since its a CatalogType list)